packages/webdriver/src/local.ts

```diff
@@ -24,7 +22,9 @@ type LocalDriverName = 'chromedriver' | 'geckodriver';
  * suopports a single Firefox session per driver process, we spawn a
  * new process for each local driver.
  */
-export function * Local(driverName: LocalDriverName, options: Options): Operation<WebDriver> {
+export function * Local(options: Options): Operation<WebDriver> {
+
+  let driverName = options.browserName === 'chrome' ? 'chromedriver' : 'geckodriver';
```

Comment:
This doesn't seem very future proof, what about Safari, Edge, etc?

Reply:
I'm not sure, but I don't think so. For local Webdriver, there is only a finite list to be supported. 'chromedriver', 'safaridriver', 'gecko driver'. When we add support for those drivers, we can resolve those too.

For remote Webdriver configs, when we land them, the `browserName` is just passed through to the Webdriver session transparently and _it_ will decide which browser process to route to.

The main goal with this change was to compress the driver options into a single object instead of two, so that they could be specified in the project config.